common/fs/src/cache/mod.rs

```diff
@@ -49,6 +49,7 @@ type FsResult<T> = Result<T, Error>;
 pub const EVENT_STREAM_BUFFER_COUNT: usize = 1000;
 const EVENT_RETRY_DELAY_MS: u64 = 1000;
 const EVENT_RETRY_COUNT: u32 = 5;
+const TAIL_WARN_THRESHOLD_B: u64 = 3000000;
```

Comment:
Two questions: One, was this value chosen arbitrarily? Two, what does the `_B` represent? Bytes?

Reply:
1 - basically, it doesn't affect functionality in anyway. It's just if we warn the user that large files are being lookback'd on
2 - yes